Actions/CheckForUpdates/CheckForUpdates.HelperFunctions.ps1

```diff
@@ -271,6 +274,9 @@ function GetWorkflowContentWithChangesFromSettings {
     $baseName = [System.IO.Path]::GetFileNameWithoutExtension($srcFile)
     $yaml = [Yaml]::Load($srcFile)
     $workflowScheduleKey = "$($baseName)Schedule"
+    $workflowBlacklist = @('UpdateGitHubGoSystemFiles', 'Troubleshooting')

```

Comment:
I would move these lines down to the section where they are used to improve readability